modules/tinymce/src/plugins/searchreplace/test/ts/browser/SearchReplacePrevNextTest.ts

```diff
@@ -17,6 +17,8 @@ describe('browser.tinymce.plugins.searchreplace.SearchReplacePrevNextTest', () =
 
   const body = SugarBody.body();
 
+  const findInputSelector = Utils.getFindInputSelector();
```

Comment:
can't this been use in place like `Utils.getReplaceWithInputSelector()` in this file?